contracts/PSM/ConstantFee.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.19;
+
+import { Fixed256x18 } from "@tempusfinance/tempus-utils/contracts/math/Fixed256x18.sol";
+import { IPSMFeeCalculator } from "./IPSMFeeCalculator.sol";
+
+/// @dev Constant fee calculator for PSM.
+contract PSMConstantFee is IPSMFeeCalculator {
+    using Fixed256x18 for uint256;
+
+    /// @dev Fee percentage for buying R from PSM.
+    uint256 public immutable buyRFee;
```

Comment:
maybe make these values changeable by owner? so there is no need to redeploy a new contract to change them. Up to you.

Reply:
Agreed. Will do it like that.